### PR TITLE
Added python workflows

### PR DESCRIPTION
**Pull request by Nikita-Smirnov-Exactpro**

### CODE REVIEW

File: .github/workflows/compound-python-matrix-test.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+name: Run matrix Python test
+
+
+on:
+  workflow_call:
+    inputs:
+      test-dir:
+        required: true
+        type: string
+        description: 'Dirrectiry with pytests'
```
Comment (davitmamrikishvili) [inline .github/workflows/compound-python-matrix-test.yml:11]:
```suggestion
        description: 'Directory with pytests'
```

File: .github/workflows/compound-python-matrix-test.yml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+      test-ignore-dir:
+        required: false
+        type: string
+        default: ''
+        description: 'Python version'
+      test-requirements-file:
+        required: false
+        type: string
+        default: ''
+        description: 'Path to reuqirments'
```
Comment (davitmamrikishvili) [inline .github/workflows/compound-python-matrix-test.yml:21]:
```suggestion
        description: 'Path to requirements'
```

File: .github/workflows/compound-python.yml [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+      strict-scanner:
+        required: false
+        type: boolean
+        default: true
+        description: 'Whether vulnerabilities and license scanning executions are strict'
+      test-dir:
+        required: false
+        type: string
+        default: ''
+        description: 'Dirrectiry with pytests'
```
Comment (davitmamrikishvili) [inline .github/workflows/compound-python.yml:38]:
```suggestion
        description: 'Directory with pytests'
```

File: .github/workflows/compound-python.yml [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+      test-ignore-dir:
+        required: false
+        type: string
+        default: ''
+        description: 'Python version'
+      test-requirements-file:
+        required: false
+        type: string
+        default: ''
+        description: 'Path to reuqirments'
```
Comment (davitmamrikishvili) [inline .github/workflows/compound-python.yml:48]:
```suggestion
        description: 'Path to requirements'
```

File: .github/workflows/python-scan.yml [hunk 5]
```diff
@@ -9,12 +9,12 @@ env:
 
 jobs:
   scan:
+    name: Vulnerability scan
     runs-on: ubuntu-latest
     steps:
       - name: Check out master
```
Comment (davitmamrikishvili) [inline .github/workflows/python-scan.yml:15]:
```suggestion
      - name: Checkout
```

File: .github/workflows/python-test.yml [hunk 6]
```diff
@@ -0,0 +1,58 @@
+name: Run matrix Python test
+
+
+on:
+  workflow_call:
+    inputs:
+      test-dir:
+        required: true
+        type: string
+        description: 'Dirrectiry with pytests'
```
Comment (davitmamrikishvili) [inline .github/workflows/python-test.yml:11]:
```suggestion
        description: 'Directory with pytests'
```

File: .github/workflows/python-test.yml [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+      test-ignore-dir:
+        required: false
+        type: string
+        default: ''
+        description: 'Python version'
+      requirements-file:
+        required: false
+        type: string
+        default: ''
+        description: 'Path to reuqirments'
```
Comment (davitmamrikishvili) [inline .github/workflows/python-test.yml:21]:
```suggestion
        description: 'Path to requirements'
```

**Review by davitmamrikishvili — COMMENTED** [review]
Also, I think adding newlines at the end of file is better.
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

File: .github/workflows/compound-python-matrix-licenses-check.yml [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+        description: 'Full version of distributive, with branch name etc.'
+      matrix-python-versions:
+        required: false
+        type: string
+        default: "['3.8', '3.9']"
+        description: 'list of python versions in JSON format'
+      matrix-os:
+        required: false
+        type: string
+        default: "['ubuntu-20.04']"
```
Comment (RaevskiySV) [inline .github/workflows/compound-python-matrix-licenses-check.yml:19]: Perhaps `latest` is possible?

File: .github/workflows/compound-python-matrix-test.yml [hunk 9]
```diff
@@ -0,0 +1,50 @@
...
+        description: 'Path to requirements'
+      matrix-python-versions:
+        required: false
+        type: string
+        default: "['3.8', '3.9']"
+        description: 'list of python versions in JSON format'
+      matrix-os:
+        required: false
+        type: string
+        default: "['ubuntu-20.04', 'windows-latest']"
```
Comment (RaevskiySV) [inline .github/workflows/compound-python-matrix-test.yml:30]: Perhaps `ubuntu-latest` is possible?

**Pull request merged**